CRM/Core/BAO/CustomGroup.php

```diff
@@ -652,13 +652,13 @@ protected static function validateSubTypeByEntity($entityType, $subType) {
       return $subType;
     }
     $contactTypes = civicrm_api3('Contact', 'getoptions', array('field' => 'contact_type'));
-    if ($entityType != 'Contact' && !in_array($entityType, $contactTypes['values'])) {
+    if ($entityType != 'Contact' && !array_key_exists($entityType, $contactTypes['values'])) {
```

Comment:
This line replicated/merged via eileen's #8389.

The other change is *not* replicated merged there.